pkg/helm/kube.go

```diff
@@ -20,8 +20,8 @@ import (
 	"sync"
 
 	"k8s.io/client-go/kubernetes"
-	// This is required to auth to various managed kube clusters (i.e. GKE)
-	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	// This is required to auth to gcp (i.e. GKE)
```

Comment:
Is there a reason to only use the GKE one and not pull in all of auth?

Reply:
It fails to build if I include all of them because of package conflicts with Azure. :sadpanda: